responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/profile-results/profile-results.component.ts

```diff
@@ -61,6 +63,7 @@ export class ProfileResultsComponent
   openProfile(selectedRegistrant: RegistrantProfileSearchResultModel): void {
     this.evacueeSessionService.profileId = selectedRegistrant.id;
     if (this.evacueeSearchService.evacueeSearchContext.hasShownIdentification) {
+      this.wizardStep.editProfileFromDashboard(selectedRegistrant.id);
```

Comment:
This is incorrect. We should not be setting values from edit wizard from profile results. This line needs to move to wizard activate guard